### DISCUSSION TITLE
Add a testing helper package

### DISCUSSION THREAD

**Discussion by erdtsieck** [JasperFx/marten#2904] [Ideas]
For testing projections or just store operations.

I will build the initial package myself and add examples to the docs

**Comment by jeremydmiller** [comment]
And automatically moving this to a discussion please.

**Comment by erdtsieck** [comment]
I created a Marten.TestHelpers package and added samples for testing one offs, async projections and async listeners: https://github.com/JasperFx/marten/pull/2903

This is an initial set-up of course, feedback is welcome
